Sources/eppo/UniversalFlagConfig.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+
+public struct UniversalFlagConfig : Decodable {
+    let createdAt: Date?;
+    let flags: [String: UFC_Flag];
```

Comment:
I am prefixing all models with `UFC_` to avoid collisions with RAC-models. We can drop the at the end of the project but I personally don't mind it.